src/app/view/perspectives/chart/data/convertor/chart-data.ts

```diff
@@ -28,11 +30,12 @@ export interface ChartDataSet {
   id: string;
   points: ChartPoint[];
   color: string;
-  isNumeric: boolean;
   yAxisType: ChartYAxisType;
   name: string;
   draggable: boolean;
   resourceType: ChartAxisResourceType;
+  category: ChartAxisCategory;
+  config?: ConstraintConfig;
```

Comment:
Why is constraint config duplicated here if it can be easily retrieved directly from the store?

Reply:
`chart-data-convertor` converts all data to format for chart. I think it's better to do everything in one place. Other reason is that I want chart to be separated from other components and data fetching.